coordinator/src/test/scala/filodb.coordinator/queryplanner/SingleClusterPlannerSpec.scala

```diff
@@ -2202,6 +2202,26 @@ class SingleClusterPlannerSpec extends AnyFunSpec with Matchers with ScalaFuture
       isInstanceOf[StaticFuncArgs] shouldEqual(true)
   }
 
+  it("should add le label filter correctly for histogram query with PeriodicSeriesWithWindowing logical plan") {
```

Comment:
Can you assert in the same style as we do in ``PlannerHierarchySpec``? Personally I prefer that as one can see the actual plan visually.